### ISSUE TITLE
[BUG] Materialized views not visible in explorer

### ISSUE THREAD

**Issue by dvrensk** [cweijan/vscode-database-client#285]
- OS INFO : Darwin Anguilla.local 18.7.0 Darwin Kernel Version 18.7.0: Tue Jun 22 19:37:08 PDT 2021; root:xnu-4903.278.70~1/RELEASE_X86_64 x86_64
- Database: Postgres 13

I make heavy use of materialised views, but unfortunately they don't show up under either the "Table" folder nor under the "View" folder.

## Steps to recreate

In a normal, blank Postgres DB, run

```sql
CREATE MATERIALIZED VIEW public.feature_names AS
SELECT
  feature_name
FROM
  "information_schema"."sql_features";

select count(*) from public.feature_names; -- just to check
```

## Expected result

The relation "feature_names" should be visible in one of the drawers in the database.

## Actual result

The "select count" provides numeric output, but the relation is not listed anywhere.

**Comment by cweijan** [comment]
Materialized views seem to exist independently of tables and views, and I will not consider supporting them in a short time .

**Comment by dvrensk** [comment]
Thanks for taking the time to look at it!

**Comment by cweijan** [comment]
Fix at version 4.2.3.

**Comment by dominijk** [comment]
I'm using v1.3.9 but do not see materialized view in the explorer side bar, I can select the data with SQL so it is there and visible. Do i need additional plugins or premium version?
